hack/local-up-cluster.sh

```diff
@@ -416,12 +422,22 @@ EOF
     create_client_certkey client-ca scheduler system:scheduler system:masters
     create_client_certkey client-ca admin system:admin system:cluster-admins
 
+    # Create auth proxy client ca
+    sudo /bin/bash -e <<EOF
+    rm -f "${CERT_DIR}/auth-proxy-client-ca.crt" "${CERT_DIR}/auth-proxy-client-ca.key"
+    openssl req -x509 -sha256 -new -nodes -days 365 -newkey rsa:2048 -keyout "${CERT_DIR}/auth-proxy-client-ca.key" -out "${CERT_DIR}/auth-proxy-client-ca.crt" -subj "/C=xx/ST=x/L=x/O=x/OU=x/CN=ca/emailAddress=x/"
+    echo '{"signing":{"default":{"expiry":"43800h","usages":["signing","key encipherment","client auth"]}}}' > "${CERT_DIR}/auth-proxy-client-ca-config.json"
+EOF
+    create_client_certkey auth-proxy-client-ca auth-proxy system:auth-proxy
+
+    cat "${CERT_DIR}/client-ca.crt" "${CERT_DIR}/auth-proxy-client-ca.crt" > "${CERT_DIR}/complete-client-ca-bundle.crt"
```

Comment:
ugly name. Why not drop the "complete-" prefix?

And I guess you need a sudo around that?